analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -116,6 +118,46 @@ class Conan(
         stashDirectories(directoryToStash).use {
             conanConfig?.also {
                 run("config", "install", it.absolutePath)
+
+                val remoteList = run("remote", "list", "--raw")
+                if (remoteList.isSuccess) {
+                    remoteList.stdout.lines().forEach { line ->
+                        val trimmedLine = line.trim()
+                        if (trimmedLine.isEmpty() || trimmedLine.startsWith('#')) return@forEach
+
+                        val wordIterator = trimmedLine.splitToSequence(' ').iterator()
+
+                        if (!wordIterator.hasNext()) return@forEach
+                        val remoteName = wordIterator.next()
+
+                        if (!wordIterator.hasNext()) return@forEach
+                        val remoteUrl = wordIterator.next()
+
+                        remoteUrl.toUri().onSuccess { uri ->
+                            log.info { "Found remote '$remoteName' pointing to URL $remoteUrl." }
+
+                            val auth = Authenticator.requestPasswordAuthentication(
+                                /* host = */ uri.host,
+                                /* addr = */ null,
+                                /* port = */ uri.port,
+                                /* protocol = */ uri.scheme,
+                                /* prompt = */ null,
+                                /* scheme = */ null
+                            )
+
+                            if (auth != null) {
+                                val userAuth = run("user", "-r", remoteName, "-p", String(auth.password), auth.userName)
+                                if (userAuth.isError) {
+                                    log.warn { "Failed to configure user authentication for remote '$remoteName'." }
+                                }
```

Comment:
If the authentication fails, shouldn't this log an error?
As the followup analyser run would most likely fail as well?

Reply:
Agreed, done.